packages/design-system/package.json

```diff
@@ -46,7 +46,7 @@
     "@testing-library/react": "12.1.2",
     "@testing-library/react-hooks": "^7.0.2",
     "@testing-library/user-event": "13.5.0",
-    "@types/jest": "^27.4.1",
```

Comment:
Is there a reason for this update?

Reply:
sorry about that we don't need that update